src/components/Article.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, text, date }) => (
+  <>
+    <h1>
+      { title }
+      {' '}
+    </h1>
```

Comment:
```suggestion
    <h1>
      {title}
    </h1>
```
Why do you need these spaces? 🤔